flickr.py

```diff
@@ -25,20 +25,23 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr_by_tags(some_input):
```

Comment:
Some really great stuff here! I think this might be just the code we've been looking for.